README.md

```diff
@@ -24,6 +24,48 @@ Then build the `pace` docker image at the top level.
 make build
 ```
 
+## Building bare-metal
+
+Pace requires gcc > 9.2, cuda, MPI, and Python 3.8 on your system in order to run bare-metal.
+
+You will also need the headers of the boost libraries in your `path` (boost itself does not need to be installed).
+
+```shell
+cd $BOOST_LOCATION
+wget https://boostorg.jfrog.io/artifactory/main/release/1.79.0/source/boost_1_79_0.tar.gz
+tar -xzf boost_1_79_0.tar.gz
+mkdir -p boost_1_79_0/include
+mv boost_1_79_0/boost boost_1_79_0/include/.
+setenv BOOST_ROOT $BOOST_LOCATION/boost_1_79_0
+```
+
+After cloning Pace you will need to update the repository's submodules:
+```shell
+$ git submodule update --init
+```
+
+We recommend creating a python venv or conda environment specifically for Pace.
+
+Inside of your pace `venv` or conda environment install the Python requirements and GT4Py:
+```shell
+$ pip install -r requirements.txt -c constraints.txt
+$ pip install --no-cache-dir -c ./constraints.txt "PATH/TO/PACE/pace/external/gt4py"
+python3 -m gt4py.gt_src_manager install --major-version 2
```

Comment:
There is a bit of mixing between `pip` and `python3` here. I'd suggest using `pip3` and `python3` or `pip` and `python`.